policy/modules/services/bird.te

```diff
@@ -26,17 +26,23 @@ files_pid_file(bird_runtime_t)
 # Local policy
 #
 
-allow bird_t self:capability net_admin;
+allow bird_t self:capability { net_admin net_raw };
 allow bird_t self:netlink_route_socket create_netlink_socket_perms;
 allow bird_t self:tcp_socket create_stream_socket_perms;
+allow bird_t self:unix_stream_socket { listen accept };
```

Comment:
Increase perms to `create_stream_socket_perms`. The other perms from that set are hiddin in the `logging_send_syslog_msg()` call.

Reply:
Fixed.